assets/component-slider.css

```diff
@@ -103,6 +113,17 @@ slider-component.slider-component-full-width {
   .slider.slider--tablet.contains-content-container .slider__slide {
     --focus-outline-padding: 0rem;
   }
+
+  .slider.slider--tablet.grid--collapsed .slider__slide {
+    --collapsed-offset-y: 0px;
+    --collapsed-shadow-offset-y: 0px;
+    --collapsed-shadow-offset-x: var(--border-width);
+  }
+
+  .slider.slider--tablet.grid--collapsed .slider__slide:first-child {
+    --collapsed-shadow-offset-x: 0px;
+  }
```

Comment:
I'm applying it to a `slider__slide` just because the selectors these are attempting to override is applied to `slider__slide` (`grid__item`). 

Basically, the first rule is overriding the values (from `grid-collapsed.css`) on _all_ the slides purposefully. For example.. If a slider grid has `.grid--2-col-tablet-down` then every other item would have 0px shadow offset. Since an active slider wouldn't be in a 2 column format, we want _all_ items to have the same x offset, except for the first.

This is the type of selector sliders need to override from grid-collapsed.css..
```
/* reset shadow origin along first row/col so its flush with grid edge */
  .grid--1-col-tablet-down.grid--collapsed-x .grid__item,
  .grid--2-col-tablet-down.grid--collapsed-x .grid__item:nth-child(2n + 1) {
    --collapsed-shadow-offset-x: 0px;
  }
  ```
  
Also in the corner commit it looks like I had to add even more specificity here and that prompted me to include a comment `/* certain specificity needed for all rules below, modify with care */`.